frontend/src/pages/modelServing/screens/global/InferenceServiceAPIProtocol.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { Label, Text, TextVariants } from '@patternfly/react-core';
+import { ServingRuntimeKind } from '~/k8sTypes';
+import { getAPIProtocolFromServingRuntime } from '~/pages/modelServing/customServingRuntimes/utils';
+import { ServingRuntimeAPIProtocol } from '~/types';
+
+type Props = {
+  servingRuntime?: ServingRuntimeKind;
+  isMultiModel?: boolean;
+};
+
+const InferenceServiceAPIProtocol: React.FC<Props> = ({ servingRuntime, isMultiModel }) => {
+  const apiProtocol =
+    (servingRuntime && getAPIProtocolFromServingRuntime(servingRuntime)) ?? undefined;
+
+  // If it is multi-model, we use REST as default
+  if (!apiProtocol && isMultiModel) {
+    return <Label color="gold">{ServingRuntimeAPIProtocol.REST}</Label>;
+  }
+
+  if (!apiProtocol) {
```

Comment:
Same thing here, if the apiProtocol does not conform to the values we have in the enum, just display Not defined